### PR TITLE
MNT: Adding CODEOWNERS

### PR DESCRIPTION
**Pull request by tangkong**
(autogenerated) Adding codeowners

### CODE REVIEW

**Comment by tangkong** [comment]
This was autogenerated but I want to test some behavior I can't quite find docs on

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Default owners are pcds-platforms-development
+* @pcdshub/pcds-platform-development
```
Comment (ZLLentz) [inline CODEOWNERS]: Possibly use a less disruptive team for the tests
```suggestion
* @pcdshub/test-team
```

Comment (ZLLentz) [reply CODEOWNERS]: Unless we needed to ping the whole formerly known as platforms team for our purposes here

Comment (ZLLentz) [reply CODEOWNERS]: I think we should make a test-team that just has @robert-test-tk on it

Comment (tangkong) [reply CODEOWNERS]: I'll just make test-tk the default owner here, no need to test the teams quite yet.  I just want to see what happens when we have multiple file types

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**